### PR TITLE
First release with limited functionality

### PR DESCRIPTION
**Pull request by valentin-pinkau**
This PR introduces a first release with limited functionality. 
The included features are documented in live scripts, but can roughly be described as follows:
- Exploration of the available atlases, parcellations, spaces and regions
- Fetching the reference templates for each space as a nifti
- Fetching the labelled parcellation map as a nifti or a list of niftis
- Fetching the continuous map and labelled map for each region as a nifti

### CODE REVIEW

**Comment by dickscheid** [comment]
This looks good already. The region.getMask() should not return the whole parellation map, but instead

- set all voxels to zero except for the ones labelled with the region's labelIndex, if defined, *or*
- set all voxels to zero except for the ones labeled by any child region's labelindex.

The result should bre a binary mask for the region, with the same dimensions and affine matrix as the parcellation map. It will make sense to cache both the obtained parcellation map NIfTI, *and* the computed region mask.

**Comment by dickscheid** [comment]
The base types (here: Space) contains an attribute 'NormalizedName'. In siibra-python, the same role is taken by the 'key' attribute, which is a unique, human readable, snake-case string. I assume this property is not exposed throught the API at this point, but we should keep in mind that this is the same thing and could be forwarded through the API. If it is accessible from the API, I suggest to replace.

**Comment by xgui3783** [comment]
Also some context, with respect to siibra-api, it can already accept some fuzzy names, e.g.:

```sh
curl https://siibra-api-stable.apps.hbp.eu/v1_0/atlases/human/parcellations/2.9
```

would return parcellation metadata of julich brain 2.9

region fuzzy match is a little complicated, and perhaps is only available in v2_0, e.g.

```sh
curl https://siibra-api-stable.apps.hbp.eu/v2_0/atlases/human/parcellations/2.9/regions?find=hoc1
```

**Comment by xgui3783** [comment]
Also, in `+siibra/+internal/initAtlases.m`, perhaps the API endpoints should reference `+siibra/+internal/API.m` rather than hard coded.

**Comment by SBludau** [comment]
I am having problems with the cache folder again on my windows system (Win10, Matlab 2022a).

Maybe it would be a good idea to use the matlab [filesep()](https://de.mathworks.com/help/matlab/ref/filesep.html) function in e.g. +siibra/+internal/initAtlases.m in line 
5?
>cached_file_name = ['+siibra' filesep 'cache' filesep 'atlases.mat'];

Then the path would be adapted to the own OS in each case.

I already tried to change the variable cached_file_name manually, but it still didn't work?

File: +siibra/+internal/initAtlases.m [hunk 1]
```diff
@@ -2,9 +2,10 @@
 arguments
     clear_cache (1, 1) logical
 end
-    cached_file_name = '+siibra/cache/atlases.mat';
+    cached_file_name = fullfile('+siibra', 'cache','atlases.mat');
```
Comment (valentin-pinkau) [inline +siibra/+internal/initAtlases.m:5]: @SBludau Good catch! Yes, I introduced the usage of fullfile where ever I set a path. This should give us compatibility with windows. Please try again :)

**Comment by dickscheid** [comment]
On first installation, the cache folder does not exist, leading to errors:

```
Error using save
Cannot create 'atlases.mat' because '+siibra/cache' does not exist.

Error in siibra.internal.initAtlases (line 15)
        save(cached_file_name, 'atlases');

Error in siibra.viewAtlases (line 3)
    atlases = siibra.internal.initAtlases(false);
```

Siibra should create the folder if not existent.

**Comment by dickscheid** [comment]
Here is a list issues I came across when going through the snippets:

Critical for PR:


Schedule for later:

- [ ] getParcellation in 01/02 should not require atlas specification (usually parcellation name is unique)
- [ ]

**Comment by dickscheid** [comment]
Merging this now. 

Code cleanup will be done as a PR by Valentin. 
Jülich will work on a proper README and simple walkthrough live script on main.

**Pull request merged**